data/json/items/containers.json

```diff
@@ -14,13 +14,22 @@
     "material": "plastic",
     "symbol": ")",
     "color": "dark_gray",
+    "//guinea pig": "fill out all the pocket properties to wrap my head around them, hence the excessive comments ",
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
+        "min_item_volume": "1 ml",
         "watertight": true,
         "rigid": true,
         "max_contains_volume": "2500 ml",
-        "max_contains_weight": "5 kg"
+        "max_contains_weight": "10 kg",
+        "spoil_multiplier": 0.95,
+        "//magic": "tiny weight_multiplier due to ergonomics?",
+        "weight_multiplier": 0.98,
```

Comment:
nothing in vanilla should have a weight_multiplier. this magically reduces the weight of the contents.